synadm/cli/__init__.py

```diff
@@ -33,6 +33,15 @@
 from synadm import api
 
 
+output_format_help = """The 'human' mode gives a tabular or list view depending
+on the fetched data, but often needs a lot of horizontal space to display
+correctly. 'json' returns formatted json. 'minified' is minified json, suitable
+for scripting purposes. 'pprint' shows a formatted output with the help of
+Python's built-in pprint module.  'yaml' is a compromise between human- and
+machine-readable output, it doesn't need as much terminal width as 'human' does
+and is the default on fresh installations."""
+
```

Comment:
I rewrote that. Please review again! Thanks in advance!

Reply:
Ignoring as [you already requested to review it](https://github.com/JOJ0/synadm/pull/92#discussion_r1116659101) and [your force push](https://github.com/JOJ0/synadm/compare/ba2812500f11650bcc77fd6383135f73d0a433f2..a2c2f2882aee99437337429365c56652b66f49f9) didn't seem to change anything.

Reply:
Ok, then probably I only force pushed the change with "separator" spelled wrongly. All good, then this is done :-P